### PR TITLE
fix: bug on contributors page

### LINKED ISSUE #57: Contributors page throws FormatException
**Describe the bug**
The Contributors tab throws FormatException as soon as it is opened

**To Reproduce**

1. Open App
2. Tap on the contributors tab
3. See Error

**Expected behavior**
There should be no exception thrown and the app should continue to fetch data and display it normally

**Screenshots (Optional)**

Code_h05o1W3mYI

### PR DESCRIPTION
**Pull request by Riktam-Santra**
### Fixed issue on contributors page:
[Contributors page throws FormatException](https://github.com/IEEE-VIT/hacktoberfest-flutter/issues/57)

### Changes made:
-   made Contributors page use Uri.parse rather than Uri.https to prevent FormatException.

### Linked issue:
Resolves #57

### CODE REVIEW

File: lib/screens/contributors.dart
Comment (dk-a-dev) [inline lib/screens/contributors.dart:1]: Please Revert Back only including this be only fix

**Review by dk-a-dev — CHANGES_REQUESTED** [review]
Two issues declared duplicate
One fix to be reviewed after changes

File: lib/screens/contributors.dart
Comment (Riktam-Santra) [reply lib/screens/contributors.dart:1]: Updated the PR, reverted color changes on splash_screen and info_page

NOTE: This has also reverted the change for #58

**Review by dk-a-dev — APPROVED** [review]
👍

**Pull request merged**